resources/metadata/gctsDeployCommit.yaml

```diff
@@ -0,0 +1,62 @@
+metadata:
+  name: gctsDeploy
+  description: Pulls a commit from the remote Git repository to a local repository
+  longDescription: |
+    Pulls a commit from the corresponding remote Git repository to a specified local repository on an ABAP system. If no <commit> parameter is specified, this step will pull the latest commit available on the remote repository.
+
+spec:
+  inputs:
+    secrets:
+      - name: abapCredentialsId
+        description: Jenkins credentials ID containing username and password for authentication to the ABAP system on which you want to deploy a commit
+        type: jenkins
+    params:
+      - name: username
+        type: string
+        description: User to authenticate to the ABAP system
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: true
+        secret: true
+      - name: password
+        type: string
+        description: Password to authenticate to the ABAP system
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: true
+        secret: true
+      - name: repository
```

Comment:
Would be multi-repository handling a use case in this step as well?

Reply:
To add some context: It was necessary to mass enable abapEnvironmentPullGitRepo to use it in a predefined pipeline, where it is not so easy to call the same step multiple times within one stage.

Reply:
Implementing this for this step isn't a problem. I guess it does no harm and it might be beneficial for some use cases. 
Although, maybe it would make sense to find a solution to the aforementioned problem of not being able to call the same step inside a single stage with different parameters other than adjusting every step to this fact.